### ISSUE TITLE
Bot can't send messages back

### ISSUE THREAD

**Issue by mwda** [witnessmenow/Universal-Arduino-Telegram-Bot#89]
Hi. I'm new here.
The bot is not sending messages to the chat. He receives the messages but can not answer them.

**Comment by jhmluna** [comment]
Are you using some library example?

**Comment by mwda** [comment]
Yes, i'm using the example in library caled "UsingWiFiManager". I'm able to send messeges, my ESP8266 do the what i sayd to do except send a message back.
